backend/test/helpers/auth.test.js

```diff
@@ -0,0 +1,71 @@
+const serverConfig = require('../../src/helpers/config');
+const { getBearerTokenFromRequest, verifyToken } = require("../../src/helpers/auth");
+let app;
+
+describe("Unauthorized routes.", () => {
+    beforeEach(() => {
+        app = serverConfig();
+    });
+
+    it("Returns 401 and can parse token but token is invalid.", async () => {
+        const response = await app.inject({
```

Comment:
maybe leaving a comment for the app.inject would be great, even leaving a url like this one.
https://www.fastify.io/docs/latest/Guides/Testing/